docs/courses/README.md

```diff
@@ -1,28 +1,101 @@
-# Contributing to LibreLingo courses
+# LibreLingo Course Documentation
 
-LibreLingo courses consist of files that contain course content. These files
-are organized in a specific directory structure.
-[Learn how to edit or create courses](editing-courses.md)!
+This document describes how the course system is laid out on LibreLingo and information on how to contribute to the course data.
 
+This document is the first step whether you want to edit an existing course, contribute a translation to an existing course or create a new one from scratch. This is also a good resource to read if you just want to learn about the course system in general.
+
+---
+
+**NOTE**: If you are not clear about anything on this document feel free to post on [Discussions](https://github.com/kantord/LibreLingo/discussions) so that the community can help you further.
+
+---
+
+**Table of Contents:**
+- [Navigation](#navigation)
+- [Basics](#basics)
+  - [Terminology](#terminology)
+  - [Exploring the courses](#exploring-courses)
+  - [Things to do for new contributors](#things-new-contributors)
+- [Courses currently open for contributions](#courses-open)
+  - [For English speakers](#courses-from-english)
+- [Caution](#caution)
+  - [Languages that don't use the Latin alphabet](#non-latin-alphabet)
+
+## Navigation
+
+- If you are totally new to the project, read the [Basics](#basics) section below first. It will give you a general understanding about things.
+- If you want to edit existing courses or translate them, go to [this page](editing-courses.md).
+- If you want to create new courses (typically suited for course designers or advanced users), go to [this page](creating-courses.md).
```

Comment:
```suggestion
- Want to create a new course? [Read out guide on how to create new courses.](creating-courses.md). (Advanced)
```

Reply:
Here is my suggestion with minor adjustments:
```md
- Want to create a new course? Read out [this guide on how to create new courses](creating-courses.md). (Advanced)
```